tests/experimental/datasources/conftest.py

```diff
@@ -88,7 +91,15 @@ def __init__(self, attributes):
                     for k, v in attributes.items():
                         setattr(self, k, v)
 
-            return [Row(row_dict) for row_dict in splitter_query_response]
+            # We know that splitter_query_response is non-empty because of validation
+            # at the top of the outer function.
+            # In some cases, such as in the datetime splitters,
+            # a dictionary is returned our from out splitter query with the key as the parameter_name.
+            # Otherwise, a list of values is returned.
+            if isinstance(splitter_query_response[0], dict):
```

Comment:
@billdirks Perhaps for stylistic purposes, if there is a `return` as part of the `if` clause, then the `else` is not needed -- just another `return` underneath (because it will logically follow).  Thank you.

Reply:
Done.

Reply:
🙇